subprojects/vinumc/eval.c

```diff
@@ -242,35 +253,33 @@ DO_CALLS_FUNC_SIGNATURE(do_calls_call) {
 
 		// put the extern function call return on the out str
 		struct return_value call_return = symbol_info->as.func(tmp_out.base);
-		put_str(out, call_return.ptr);
+
+		if ((flags & REDUCE_BLANKS) != 0) {
+			put_blank_reduced_str(out, call_return.ptr, true, true);
+		} else {
+			put_str(out, call_return.ptr);
+		}
```

Comment:
The vinum specification doesn't say if the output of external functions should be trimmed, so I decided make them match the behavior of regular functions.